design/design-tools/adobe-illustrator.md

```diff
@@ -0,0 +1,16 @@
+# Adobe Illustrator
+
+## Create Icons
+
+Create vector illustrations and icons which can be easily exported to Adobe XD.
+
+
+## Save for Web
+
+Optimise vector illustrations and icons to reduce the file size and improve page load speeds.
+
+
+## Social Media Posts
+
+Use Illustrator as an alternative to Adobe Spark for more complex and unique designs.
+- [Social media post resolutions](design-tools/social-media.md)
```

Comment:
I think this link is broken?

```suggestion
- [Social media post resolutions](social-media.md)
```

Reply:
Fixed :)

Reply:
you should be able to click "commit suggestion" and it'll automatically commit it for you 👍